packages/gcp/src/domain/GcpFootprintEstimationConstants.ts

```diff
@@ -140,7 +140,7 @@ export const GCP_CLOUD_CONSTANTS: CloudConstantsByProvider = {
   SERVER_EXPECTED_LIFESPAN: 35040, // 4 years in hours
 }
 
-const getGCPEmissionsFactors = (): CloudConstantsEmissionsFactors => {
+export const getGCPEmissionsFactors = (): CloudConstantsEmissionsFactors => {
```

Comment:
Let me know if I'm missing something, but this looks like a general refactor. I like it, but unless it's needed for the backstage plugin, I think a separate PR might be better. One reason is your branch still has a reference to `GCP_EMISSIONS_FACTORS_METRIC_TON_PER_KWH` in some test code, and I'd like to make sure the change is applied consistently.

Reply:
Same reason as above ☝️

Reply:
We'll update the other instances of `GCP_EMISSIONS_FACTORS_METRIC_TON_PER_KWH` (I thought we got them all!).

Reply:
Not sure here - do you refer to `GCP_EMISSIONS_FACTORS_METRIC_TON_PER_KWH` in `App.test.ts`?

Reply:
Yeah, that is the place I'm referring to. Instead of mocking the removed constant, could we mock the return value for the new `getGCPEmissionsFactors` to be the same values: 

```gcpRegion1: 3,
    gcpRegion2: 4,```